service/controller/app/v1/key/key_test.go

```diff
@@ -5,7 +5,7 @@ import (
 	"testing"
 
 	"github.com/giantswarm/apiextensions/pkg/apis/application/v1alpha1"
-	"k8s.io/apimachinery/pkg/apis/meta/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
Fixing the import name. 

@tomahawk28 FYI. We use this naming convention to avoid conflicts when we need multiple k8s packages.